### ISSUE TITLE
Support Svelte 4

### ISSUE THREAD

**Issue by cloudlena** [kaisermann/svelte-i18n#216]
**Is your feature request related to a problem? Please describe.**

It would be awesome to support Svelte 4. Currently, I get the following error when trying to install `svelte@4` in a project with `svelte-i18n`:

```txt
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: cloudnativeday.ch@undefined
npm ERR! Found: svelte@4.0.0
npm ERR! node_modules/svelte
npm ERR!   dev svelte@"^4.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"^3.25.1" from svelte-i18n@3.6.0
npm ERR! node_modules/svelte-i18n
npm ERR!   dev svelte-i18n@"^3.6.0" from the root project
``` 

**Describe the solution you'd like**

Please update the peer dependencies to include Svelte 4.

**Comment by kaisermann** [comment]
Hey @cloudlena 👋  thanks for the issue. I will be releasing a new version with the updated peer dependency soon 🙏

**Comment by kaisermann** [comment]
Should be released as `v3.7.0`
